app/src/main/java/com/andela/helpmebuy/fragments/RequestFragment.java

```diff
@@ -77,7 +97,13 @@ public void onClick(View v) {
         user = CurrentUserManager.get(getContext());
     }
 
+
+    private boolean isConnectionAccepted(Connection connection) {
+        return connection.getConnectionStatus() == ConnectionStatus.ACCEPTED.getStatus();
+    }
+
     private void loadConnections() {
+        progressWheel.spin();
```

Comment:
NITPICK: Why not maintain the message display method we used in the contact fragment.